<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInfo</class>
 <widget class="QDialog" name="EmployeeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>151</y>
     <width>180</width>
     <height>97</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_Save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Delete">
      <property name="text">
       <string>Hapuuus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TableView</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sec_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>601</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>225</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_EID">
        <property name="text">
         <string>EID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_EID"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_Surename">
        <property name="text">
         <string>Surename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_Surename"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_Name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_Name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_Age">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_Age"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ListView</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ComboBox</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>131</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Load">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>571</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaaad List</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
